activestorage/lib/active_storage.rb

```diff
@@ -67,6 +67,8 @@ module ActiveStorage
   mattr_accessor :replace_on_assign_to_many, default: false
   mattr_accessor :track_variants, default: false
 
+  mattr_accessor :video_preview_arguments, default: ["-y", "-vframes", "1", "-f", "image2", "-"]
```

Comment:
Does an array make sense here? If i'm trying to override this behavior I would expect to pass a single string as the arguments. 🤔

Reply:
Yea it doesn't really make sense. It's only this way because the original was like this for some reason. I'll switch it to a string. There's some more discussion here: #42471.